test/lib/parameters/caster_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require "parameters/caster"
```

Comment:
These should be removed, ditto in the test below.  Autoloading should handle it when the classes are moved under foreman/parameters/ to match their namespace.